docs/flink_job_cluster_guide.md

```diff
@@ -0,0 +1,95 @@
+# Running Flink-Job-Cluster in Kubernetes
+
+## Overview
+
+to run a Flink-Job-Cluster, you first have to deploy the Flink-Operator. For that, follow the [user_guide](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/master/docs/user_guide.md)
+
+## Prerequisites
+
+This Chart requires you to have Flink Operator Running in your Cluster.
+It is advised to use at least the following image version of flink-operator: ```gcr.io/flink-operator/flink-operator:v1beta1-6```
+since it enables you to use annotations for the Flink-Job-Cluster components e.g. JobManager, TaskManager and Job.
+
+## Installing the Chart
+
+The instructions to install the Flink Job Cluster chart:
+
+1. Prepare a Flink operator image. You should use at least ```gcr.io/flink-operator/flink-operator:v1beta1-6```.
+
+2. Deploy the Flink Operator Helm Chart, follow these [instructions](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/helm-chart/flink-operator)
+
+3. Prepare a Flink-Job-Cluster image. You should use ```flink:latest``` or ```flink:1.9.3``` which has been tested and confirmed to work with flink-job-cluster helm chart.
+
+3. Use the following command to install the flink-job-cluster chart:
+
+  ```bash
+  helm template --namespace [NAMESPACE] [RELEASE_NAME] [CHART] -f values.yaml |kubectl apply -f -
+  ```
+
+## Uninstalling the Chart
```

Comment:
Add a section for checking the release status.

Reply:
Hello! I was not entirely sure what you meant with "release status" so I added a section to check the current Flink Operator Image Releases. If you meant the helm-chart release status, then that wouldn't be possible as the guide is meant to deploy the Flink job cluster with the helm template command and use a piped kubectl apply at the end. This way, we only use the templating of helm but not its releases.

Reply:
Gotcha. I meant to say Helm Chart release status, now I see it is not deployed directly.